lib/screen/book_screen.dart

```diff
@@ -33,12 +36,101 @@ class _BookScreenState extends State<BookScreen> {
   Widget _appBar(BuildContext context) => SliverAppBar(
     floating: true,
     snap: true,
-    title: Text("Home",
-        style: TextStyle(
-            fontWeight: FontWeight.bold,
-            fontSize: 25,
-            color: Theme.of(context).accentColor
-        )
+    toolbarHeight: 250,
+    title: Column(
+      mainAxisAlignment: MainAxisAlignment.spaceAround,
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: <Widget>[
+        /*CircleAvatar(
+          backgroundColor: kNewAccentColor,
+          minRadius: 5.0,
+          child: CircleAvatar(
+            radius: 15.0,
+            backgroundImage: AssetImage('assets/images/avatar.png'),
+          ),
+        ),*/
```

Comment:
I was not able to position this to left or right so I left it commented out for now until we figure it out